frontend/src/pages/home/aiFlows/ProjectsGallery.tsx

```diff
@@ -6,16 +6,46 @@ import useIsAreaAvailable from '~/concepts/areas/useIsAreaAvailable';
 import useServingPlatformStatuses from '~/pages/modelServing/useServingPlatformStatuses';
 import InfoGallery from './InfoGallery';
 
+const projectDescriptionLeadText =
+  'Projects allow you and your team to organize and collaborate on resources within separate namespaces.';
```

Comment:
🤔 why not inline this -- it's only used one... you inline other text... am I missing something here?

Reply:
I had it all differently then refactored a bit and forgot to inline this